### ISSUE TITLE
Writing missing DateTime values is busted

### ISSUE THREAD

**Issue by marklio** [marklio/LinqToStdf#4]
We tried to pretend that DateTime fields were just normal fields, but now, when we generate the missing field contengincy code, we try to generate a Ldc<DateTime>, which doesn't work. Unfortunately, this needs special casing in order to generate the proper encoding.

**Comment by DesertDev702** [comment]
Hi, need your guidance. I'm using VS Studio Community 2019 to examine your project on an STD file and when I run the example app, I get this error which I'm wondering is related to what you posted? System.NotSupportedException: 
Ldc<T> does not support T is System.DateTime

**Comment by marklio** [comment]
Yeah, that looks like at least related. Are you writing STDF files, or reading them? From your post, it sounds like you're reading them. Do you have a repro? I haven't worked in the semicon industry in 15 years, so I have trouble getting traction on issues like this.

**Comment by DesertDev702** [comment]
Thanks for replying. Would like to be able to just read the binary STDF file, and then output to a human readable columnar data format, i.e. csv, comma delimited.  No, I don't have a repro.

**Comment by DesertDev702** [comment]
Would you be open and willing to do a small contract job getting back into your code and get it working as a managed code DLL for a C# or VB.Net client? Please let me know as soon as possible as this is urgent. Thanks.

**Comment by marklio** [comment]
I am not opposed to doing something like that. I'd want the goals to be clear, and I would want to have full access to the files that don't work. drop me a note at mark@marklio.com
